app/src/main/java/in/ac/iitb/gymkhana/iitbapp/fragment/MyEventsFragment.java

```diff
@@ -44,7 +71,85 @@ public void onClick(View v) {
             }
         });
         return view;
+    }
+
+    @Override
+    public void onStart() {
+        super.onStart();
+        session = new SessionManager(getContext());
```

Comment:
SessionManager is not to be referenced from inside a fragment. Whenever user details or login details are required, they're passed as arguments to the fragment. There is a lot of sample code to do this in this project itself.